src/libImaging/GetBBox.c

```diff
@@ -94,21 +96,24 @@ ImagingGetProjection(Imaging im, UINT8* xproj, UINT8* yproj)
     #define GETPROJ(image, mask)\
         for (y = 0; y < im->ysize; y++) {\
             has_data = 0;\
-            for (x = 0; x < im->xsize; x++)\
+            for (x = 0; x < im->xsize; x++) {\
                 if (im->image[y][x] & mask) {\
                     has_data = 1;\
                     xproj[x] = 1;\
                 }\
-                if (has_data)\
-                    yproj[y] = 1;\
-                }
+            }\
+            if (has_data) {\
+                yproj[y] = 1;\
+            }\
+        }
```

Comment:
Just a note.

Before:
Visually, both `if` blocks are indented as if they are both in the `for` loop.

But it wouldn't make sense for both to be in the `for` loop, so it was functionally correct before, and the change makes it clearer.